lib/phoenix_token_auth/mailer.ex

```diff
@@ -25,6 +25,7 @@ defmodule PhoenixTokenAuth.Mailer do
   Both config fields have to be functions returning binaries. welcome_email_subject receives the user and
   welcome_email_body the user and confirmation token.
   """
+  def send_welcome_email(user = %{email: email}, confirmation_token) when email == nil do end
```

Comment:
oh i thought we changed that in the users_controller and moved to two function bodies. i think i would actually like the approach with the two function bodies in users_controller better. the main reason is, that it's more explicit that way and `send_welcome_email` still raises an error when the user has no email, because of a bug or something,
would you change that or are you too annoyed already? :wink: